src/sirplugins.c

```diff
@@ -343,10 +343,10 @@ sirpluginid _sir_plugin_cache_add(sir_plugincache* spc, sir_plugin* plugin) {
     if (!_sir_validptr(spc) || !_sir_validptr(plugin))
         return 0U;
 
-    if (spc->count >= SIR_MAXPLUGINS) {
+    if (spc->count >= SIR_MAXPLUGINS) { // GCOVR_EXCL_START
```

Comment:
samesies

Reply:
💀 I guess you could try to load 625282 plug-ins?

Reply:
I have no problem loading 17 plugins over pretending we have coverage here.

Reply:
Will remove for now then.